demisto_sdk/commands/validate/file_validator.py

```diff
@@ -584,10 +588,11 @@ def is_valid_structure(self):
                 self.validate_all_files()
         else:
             if self.file_path:
-                self.branch_name = self.get_current_working_branch()
-                self.validate_committed_files()
+                print('Not using git, validating file: {}'.format(self.file_path))
+                self.is_backward_check = False  # if not using git, no need for BC checks
+                self.validate_added_files({self.file_path}, find_type(self.file_path))
```

Comment:
for keyword argument, better not to depends on order
```suggestion
                self.validate_added_files({self.file_path}, file_type=find_type(self.file_path))
```